src/open_inwoner/scss/components/File/File.scss

```diff
@@ -31,8 +31,18 @@
   &__data {
     display: flex;
     flex-direction: column;
+    adding: var(--spacing-large);
```

Comment:
Typo probably? I see padding below, so I think this can be removed

Reply:
Oops... this should not be approved :-D since there is another padding statement a couple of lines lower. I'll fix this.
There were multiple statements for this very same class in multiple places so I wanted to group them all together here.

Reply:
I have fixed this :-D